### ISSUE TITLE
[req]:  Tabbed searches

### ISSUE THREAD

**Issue by wellloaded** [Jackett/Jackett#14875]
### Is there already a request for your feature?

- [X] I have checked older issues, open and closed

### Is your feature request related to a problem? Please describe.

I like the current Jackett interface but I think there are also lot of simple improvements to it. WHen I'm in "search" tack I end up opening multiple browser tabs to run parallel searches as I have lots of indexes configured and I don't want to waste time performing 1 search at the time.

### Describe the solution you'd like

Long story short I believe it would be a great idea to have each Jackett search appearing into an (app) tab. This allows to: retain history until the tab is closed, prevent the need for multiple browser tabs and in general it's just an elegant solution for the UI.

I suppose each tab should take the name of the search performed.

### Describe alternatives you've considered

Opening multiple browser tabs but this is ugly.

**Comment by garfield69** [comment]
As none of the current team are GUI experts, we are reliant on a volunteer to contribute, and as we already have several outstanding enhancement requests for GUI open in the last several years, I do not expect a swift resolution to this one.
